net-mail/cyrus-imapd/cyrus-imapd-3.2.2.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools flag-o-matic pam ssl-cert
+
+DESCRIPTION="The Cyrus IMAP Server"
+HOMEPAGE="https://www.cyrusimap.org/"
+SRC_URI="https://github.com/cyrusimap/${PN}/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="BSD-with-attribution GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="afs backup calalarm caps clamav http kerberos ldap \
+	mysql nntp pam perl postgres replication +server sieve snmp \
+	sqlite ssl static-libs tcpd xapian"
+
+CDEPEND="
+	sys-libs/zlib:0=
+	dev-libs/libpcre:3
+	>=dev-libs/cyrus-sasl-2.1.17:2
+	dev-libs/jansson
+	dev-libs/icu:0=
+	sys-libs/e2fsprogs-libs
+	afs? ( net-fs/openafs )
+	calalarm? ( dev-libs/libical:0= )
+	caps? ( sys-libs/libcap )
+	clamav? ( app-antivirus/clamav )
+	http? ( dev-libs/libxml2:2 dev-libs/libical:0= net-libs/nghttp2 )
+	kerberos? ( virtual/krb5 )
+	ldap? ( net-nds/openldap )
+	mysql? ( dev-db/mysql-connector-c:0= )
+	nntp? ( !net-nntp/leafnode )
+	pam? (
+			sys-libs/pam
+			>=net-mail/mailbase-1
+		)
+	perl? ( dev-lang/perl:= )
+	postgres? ( dev-db/postgresql:* )
+	snmp? ( >=net-analyzer/net-snmp-5.8-r3:0= )
+	ssl? ( >=dev-libs/openssl-1.1.1g:0=[-bindist] )
+	sqlite? ( dev-db/sqlite:3 )
+	tcpd? ( >=sys-apps/tcp-wrappers-7.6.28
+		snmp? ( net-analyzer/net-snmp:0=[tcpd=] )
+		)
+	xapian? ( >=dev-libs/xapian-1.4.14:0= )
+"
+# Tests now always required
+DEPEND="${CDEPEND}
+	dev-util/cunit
+"
+
+# file collision with app-arch/dump - bug 619584
+RDEPEND="${CDEPEND}
+	acct-group/mail
+	acct-user/cyrus
+	!app-arch/dump
+"
+
+REQUIRED_USE="
+	afs? ( kerberos )
+	backup? ( sqlite )
+	calalarm? ( http )
+	http? ( sqlite )
+"
+
+src_prepare() {
+	default
+	# Fix master(8)->cyrusmaster(8) manpage.
+	for i in `grep -rl -e 'master\.8' -e 'master(8)' "${S}"` ; do
+		sed -i -e 's:master\.8:cyrusmaster.8:g' \
+			-e 's:master(8):cyrusmaster(8):g' \
+			"${i}" || die "sed failed" || die "sed failed"
+	done
+	mv man/master.8 man/cyrusmaster.8 || die "mv failed"
+	sed -i -e "s:MASTER:CYRUSMASTER:g" \
+		-e "s:Master:Cyrusmaster:g" \
+		-e "s:master:cyrusmaster:g" \
+		man/cyrusmaster.8 || die "sed failed"
```

Comment:
This is hacky. Isn't there a better way? What good does the grep do here? You should know what files need sedding and what does not.